st2common/tests/unit/services/test_workflow.py

```diff
@@ -44,8 +46,16 @@
 TEST_PACK = 'orquesta_tests'
 TEST_PACK_PATH = st2tests.fixturesloader.get_fixtures_packs_base_path() + '/' + TEST_PACK
 
+PACK_7 = 'dummy_pack_7'
+PACK_7_PATH = st2tests.fixturesloader.get_fixtures_packs_base_path() + '/' + PACK_7
+
+PACK_20 = 'dummy_pack_20'
+PACK_20_PATH = st2tests.fixturesloader.get_fixtures_packs_base_path() + '/' + PACK_20
+
```

Comment:
Can we use the pack `orquesta_tests` as one of the pack here?  Also, why dummy_pack_7 and dummy_pack_20?  We can't use the other dummy_packs like dummy_pack_1?

Reply:
Yes,  we can move dummy_pack_20 to orquesta_test2. 
The reason I used two dummy pack because I was concerning about integration test. 
When install dummy_pack_1, there is a error message `Failed to register policies: Failed to register policy "/opt/stackstorm/packs/dummy_pack_1/policies/policy_2.yaml" from pack "dummy_pack_1": Referenced policy_type "action.mock_policy_error" doesnt exist` 

I almost ran all dummy packs and found these two.